requirements/requirements.in

```diff
@@ -43,8 +43,8 @@ click
 
 # Examples
 # torch at 1.13.0 or later gives trouble with cuda dependencies
```

Comment:
This comment is now outdated. Please remove.

Reply:
done